src/odemis/acq/fastem.py

```diff
@@ -320,13 +320,18 @@ def run(self):
         # Update the position of the first tile.
         self._pos_first_tile = self.get_pos_first_tile()
 
-        # Move the stage to the first tile, to ensure the autofocus and image translation pre-align are
-        # done in the correct location and the correct position is stored in the megafield metadata yaml file.
-        self.move_stage_to_next_tile()
-
         if self._pre_calibrate:
+            # Move the stage to the tile with index (-1, -1), to ensure the autofocus and image translation pre-align
+            # are done to the top left of the first field, outside the region of acquisition to limit beam damage.
+            self.field_idx = (-1, -1)
+            self.move_stage_to_next_tile()
             self.pre_calibrate()
 
+        # Move the stage to the first tile, to ensure the correct position is
```

Comment:
Does it not make sense to move line 320/321 here but outside of the if statement? Now you move to the first tile of the ROA, then you move outside of the ROA for pre-calibration and then back to the first tile. Instead you could only move to the (-1, -1) directly and then to (0,0). That would save one step I think.

Reply:
Line 321 does not move the stage, it just calculates the position of the first tile. To move to (-1, -1) the position of the first tile needs to have been calculated.

Reply:
Ah right this was needed as `move_stage_to_next_tile` needs to know the position of the first tile.